samples/apps/csv-data-chat-webapp-streamlit/app.py

```diff
@@ -0,0 +1,161 @@
+import streamlit as st
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import semantic_kernel as sk
+import re
+import traceback
+import asyncio
+
+from semantic_kernel.connectors.ai.open_ai import AzureChatCompletion
+
+from io import StringIO
+
+async def main():
+    st.header("Streamlit Semantic Kernel Chat App 🤖")
+    # Initialize the Semantic Kernel
+    kernel = sk.Kernel()
+
+    deployment, api_key, endpoint = sk.azure_openai_settings_from_dot_env()
+    service_id = "aoai_chat_completion"
+    kernel.add_service(
+        AzureChatCompletion(service_id=service_id, deployment_name=deployment, endpoint=endpoint, api_key=api_key),
+    )
+
+    csvDataFrame = '''Name,Age,Occupation
+    John,34,Engineer
+    Jane,28,Doctor
+    Alice,30,Teacher
+    Bob,45,Manager
+    Charlie,25,Student
+    '''
+
+    # Import the necessary skills
+    skills_directory = "plugins"
+
+    interpret_skill =   kernel.import_plugin_from_prompt_directory(skills_directory, "Interpret")
```

Comment:
Based on changes from when this PR was created, the `import_plugin_from_prompt_directory` was changed to `add_plugin`.